mon-pix/app/templates/components/user-logged-menu.hbs

```diff
@@ -1,21 +1,21 @@
-{{!-- template-lint-disable no-implicit-this no-action no-curly-component-invocation --}}
-{{#if currentUser.user}}
+{{!-- template-lint-disable no-action--}}
```

Comment:
**question:** Pourquoi laisser le template-lint ?

Reply:
Regarder this.closeWindow